docs/content/users/snippets/tasks.json

```diff
@@ -0,0 +1,17 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Ddev: Enable Xdebug",
```

Comment:
Same here change `Ddev`to `DDEV. applies also to the label for disabling here

Reply:
Fixed with https://github.com/ddev/ddev/pull/5771/commits/2ee27b02e05c0a6b82d11c647eb6e968ef12e011